.github/workflows/update-shadeform-catalog.yml

```
@@ -0,0 +1,66 @@
+name: "update-shadeform-catalog"
+on:
+  schedule:
+    - cron: '0 */1 * * *'    # Every 1 hour
+  workflow_dispatch:
+
+jobs:
+  update_shadeform_catalog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone SkyPilot repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          repository: skypilot-org/skypilot
+          path: sky
+      - name: Clone Catalog repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          path: catalogs
+          token: ${{ secrets.GH_ACTION_PAT }}
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: "latest"
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          uv venv --seed ~/catalogs-venv
+          source ~/catalogs-venv/bin/activate
+          cd sky
+          uv pip install requests
+      - name: Run fetch_shadeform
+        id: fetch_catalogs
+        run: |
+          source ~/catalogs-venv/bin/activate
+          versions=$(cat catalogs/.metadata.yml | yq -r '.version_commit | keys[]')
+          for version in $versions; do
+            commit_hash=$(cat catalogs/.metadata.yml | yq -r ".version_commit.$version")
```

We should skip the version v7 for shadeform, considering it does not have the fetch_shadeform.py file in that commit?
```suggestion
            if [[ "$version" == "v7" ]]; then
                continue  # skip, as shadeform was not added by the version
            fi
            commit_hash=$(cat catalogs/.metadata.yml | yq -r ".version_commit.$version")
```